### ISSUE TITLE
Users - Table search components lack a visible label

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1132]
"Description: On the 'Browse Users' page, users are able to interact with a table of users. Associated with this table, there is a search edit field, three radio buttons, and a 'Search users' button that are able to filter the results in the table. However, the edit field and radio buttons currently lack a visible label for the individual components or for the group as a whole. 

Visually, users might be able to infer that the 'Search Users' button is acting as the visible label in this case, however as that button is not directly next to the edit field, it could be more clear to allow for more users to understand the purpose of each component. Additionally, the radio button grouping lack a group label of 'Search By', 'Options', or similar. 

Expected Results: The 'Search Users' edit box has a persistent visible label that matches the component's accessible name. The three radio buttons 'Username', 'Display Name', and 'Email' would have a visible group label that accurately describes their intended function, such as 'Search By:' or 'Options:'. 

Actual Results: The search edit field and radio buttons associated with the 'Users' table lack visual labels."

3.3.2 Labels or Instructions; Level A; WCAG 2.0
